producer.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+# thoth-package-releases-job-producer
+# Copyright(C) 2018, 2019, 2020 Bissenbay Dauletbayev, Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Check for new releases on Python index and mark new packages in the graph database.
+
+Produces messages regarding new releases for packages registered in the graph database.
+"""
+
+import logging
+import random
+import os
+import asyncio
+
+from thoth.common import init_logging
+from thoth.common import OpenShift
+from prometheus_client import CollectorRegistry, Gauge
+
+from thoth.messaging import MessageBase
+from thoth.messaging.unresolved_package import UnresolvedPackageMessage
+from thoth.messaging.unrevsolved_package import UnrevsolvedPackageMessage
+from thoth.common import __version__ as __common__version__
+from thoth.storages import __version__ as __storage__version__
+from thoth.messaging import __version__ as __messaging__version__
+from thoth.storages import GraphDatabase
+from version import __version__
+
+__service_version__ = f"{__version__}+storage.{__storage__version__}.common.{__common__version__}.messaging.{__messaging__version__}"  # noqa: E501
```

Comment:
This is already present in app.py

Reply:
`version.py`

Reply:
Thanks, @fridex , yes @pacospace I created a separate file for that. Do you want to track the `python` version as well?